examples/04-many-to-many-relationship/src/join-with-metadata/user-role.resolver.ts

```diff
@@ -32,9 +32,9 @@ export class UserRoleResolver extends BaseResolver<UserRole> {
   }
 
   @Query(() => [UserRole])
-  async userRoles(@Args() { where, orderBy, limit, offset }: UserRoleWhereArgs): Promise<
-    UserRole[]
-  > {
+  async userRoles(
+    @Args() { where, orderBy, limit, offset }: UserRoleWhereArgs
+  ): Promise<UserRole[]> {
```

Comment:
Do you know why these changes are happening? Looking to minimize this diff if possible. Can you try to remove whitespace-only changes?

Reply:
Took me a while to figure this one out, but looks like it is because of Prettier's version. The master branch resolves to [1.18.2](https://github.com/goldcaddy77/warthog/blob/master/yarn.lock#L7507), while my branch resolved to [1.19.1](https://github.com/thiagozf/warthog/blob/refactor/monorepo/yarn.lock#L7516) after my first dependencies review to declare them in the right sub-packages. Using version 1.18.2 "fixed" it. Should we change the dependency to "match exactly" (` ` instead of `^`) Prettier's version 1.18.2 to prevent it from happening again in the future?